pkg/rpcmiddleware/discover/discover.go

```diff
@@ -28,7 +28,8 @@ import (
 
 var errAddress = errors.DefineInvalidArgument("address", "invalid address")
 
-func defaultPort(target string, port int) (string, error) {
+// WithDefaultPort appends port if target does not already have one.
+func WithDefaultPort(target string, port int) (string, error) {
```

Comment:
`With` is reserved for options or funcs that change state

Reply:
Renamed to the original `DefaultPort`